src/vos/vos_common.c

```diff
@@ -143,6 +146,9 @@ vos_xsctxt_get(void)
 #ifdef VOS_STANDALONE
 	return self_mode.self_xs_ctxt;
 #else
+	if (dss_tls_get() == NULL)
```

Comment:
Minor: better to add some nice comment here.

Reply:
ok